chef.gemspec

```diff
@@ -55,6 +55,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "proxifier", "~> 1.0"
 
+  s.add_dependency "aws-sdk-s3", "~> 1.91"
```

Comment:
can you add a comment inline here saying what feature this enables similar to the hashi comment below